library/Garden/Web/Cookie.php

```diff
@@ -235,16 +237,6 @@ public function makeCookieHeader() {
         return $this->cookieEncode($this->cookies);
     }
 
-    /**
-     * Return the cookies that will be set in a format suitable for "Set-Cookie" HTTP headers.
-     *
-     * @return array Returns an array of "Set-Cookie" header values.
-     * @throws \Exception TODO.
-     */
-    public function makeSetCookieHeader() {
-        throw new \Exception('Not implemented.', 501);
-    }
-
```

Comment:
This has been in a todo state for a long time. I would rather just remove it since it doesn't seem to be needed.